modules/apps/dataset/dataset-view-service/service.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0"?>
+<!DOCTYPE service-builder PUBLIC "-//Liferay//DTD Service Builder 7.4.0//EN" "http://www.liferay.com/dtd/liferay-service-builder_7_4_0.dtd">
+
+<service-builder auto-import-default-references="false" auto-namespace-tables="false" dependency-injector="ds" mvcc-enabled="true" package-path="com.liferay.dataset.view">
+	<namespace>DatasetView</namespace>
+	<entity local-service="true" name="DatasetViewActiveEntry" remote-service="false" trash-enabled="false" uuid="true">
+
+		<!-- PK fields -->
+
+		<column container-model="true" name="datasetViewActiveEntryId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" uad-anonymize-field-name="fullName" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="datasetDisplayId" type="String" />
+		<column name="datasetViewStateEntryId" type="long" />
+		<column name="plid" type="long" />
+		<column name="portletId" type="String" />
+
+		<!-- Order -->
+
+		<!-- Finder methods -->
+
+		<finder name="U_D_P_P" return-type="DatasetViewActiveEntry" unique="true">
+			<finder-column name="userId" />
+			<finder-column name="datasetDisplayId" />
+			<finder-column name="plid" />
+			<finder-column name="portletId" />
+		</finder>
+
+		<!-- References -->
+
+	</entity>
+	<entity local-service="true" name="DatasetViewStateEntry" remote-service="false" trash-enabled="false" uuid="true">
+
+		<!-- PK fields -->
+
+		<column container-model="true" name="datasetViewStateEntryId" primary="true" type="long" />
+
+		<!-- Audit fields -->
+
+		<column name="companyId" type="long" />
+		<column name="userId" type="long" />
+		<column name="userName" type="String" uad-anonymize-field-name="fullName" />
+		<column name="createDate" type="Date" />
+		<column name="modifiedDate" type="Date" />
+
+		<!-- Other fields -->
+
+		<column name="json" type="String" />
```

Comment:
Wouldn't be these get a name closer to the intent/meaning of the column (e.g "viewState"), rather than the format?

Reply:
I copied that from other entity, but yes, you are right.TBH, I don't know what has more possibilities of passing the final review...

Reply:
Done.